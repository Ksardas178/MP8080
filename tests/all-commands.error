line 6: <WARNING> argument adapted to match 3 8-bit value(s)
line 7: <ERROR> expected 3 argument(s)
line 7: <WARNING> argument adapted to match 3 8-bit value(s)
line 8: <ERROR> expected 3 argument(s)
line 8: <WARNING> argument adapted to match 3 8-bit value(s)
line 9: <ERROR> expected 3 argument(s)
line 9: <WARNING> argument adapted to match 3 8-bit value(s)
line 10: <ERROR> expected 3 argument(s)
line 11: <WARNING> argument adapted to match 1 8-bit value(s)
line 11: <WARNING> argument adapted to match 1 8-bit value(s)
line 11: <ERROR> expected 1 argument(s)
line 11: <ERROR> expected 1 argument(s)
