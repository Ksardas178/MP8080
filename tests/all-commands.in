MOV B C
MOV M D
MOV D M

MVI D 2
MVI M 4

LXI B 8 13
LXI D 16 28
LXI H 32 34
LXI SP 63 64

SUI 69

//TODO: write all commands from table