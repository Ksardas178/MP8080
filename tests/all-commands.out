
[32;1mInput is correct
[0m
[30;1mCode analysis results:
[0m______________________________
00000000: MOV 0 1
00000001: MOV 6 2
00000010: MOV 2 6
00000011: MVI 2 2
00000100: MVI 6 4
00000101: LXI 0 8 13
00000110: LXI 2 16 28
00000111: LXI 4 32 34
00001000: LXI 6 63 64
00001001: SUI 69
